CefSharp.Core/Internals/CefFrameWrapper.h

```diff
@@ -9,271 +9,198 @@
+            // browser process.
+            ///
+            /*--cef()--*/
+            virtual void ViewSource();
+
+            ///
+            // Retrieve this frame's HTML source as a string sent to the specified
+            // visitor.
+            ///
+            /*--cef()--*/
+            virtual void GetSource(IStringVisitor^ visitor);
+
+            ///
+            // Retrieve this frame's display text as a string sent to the specified
+            // visitor.
+            ///
+            /*--cef()--*/
+            virtual void GetText(IStringVisitor^ visitor);
+
+            // TODO: Do we need this?
+            ///
+            // Load the request represented by the |request| object.
+            ///
+            /*--cef()--*/
+            //virtual void LoadRequest(CefRequestWrapper^ request)
+            //{
+            //    _frame->LoadRequest(request->GetCefRequest().get());
+            //}
+
+            ///
+            // Load the specified |url|.
+            ///
+            /*--cef()--*/
+            virtual void LoadUrl(String^ url);
+
+            ///
+            // Load the contents of |html| with the specified dummy |url|. |url|
+            // should have a standard scheme (for example, http scheme) or behaviors like
+            // link clicks and web security restrictions may not behave as expected.
+            ///
+            /*--cef()--*/
+            virtual void LoadHtml(String^ html, String^ url);
+
+            ///
+            // Execute a string of JavaScript code in this frame. The |script_url|
+            // parameter is the URL where the script in question can be found, if any.
+            // The renderer may request this URL to show the developer the source of the
+            // error.  The |start_line| parameter is the base line number to use for error
+            // reporting.
+            ///
+            /*--cef(optional_param=script_url)--*/
+            virtual void ExecuteJavaScriptAsync(String ^code, String^ scriptUrl, int startLine);
+
+            virtual Task<JavascriptResponse^>^ EvaluateScriptAsync(String^ script, Nullable<TimeSpan> timeout);
+
+            ///
+            // Returns true if this is the main (top-level) frame.
+            ///
+            /*--cef()--*/
+            virtual bool IsMain();
+
+            ///
+            // Returns true if this is the focused frame.
+            ///
+            /*--cef()--*/
+            virtual bool IsFocused();
+
+            ///
+            // Returns the name for this frame. If the frame has an assigned name (for
+            // example, set via the iframe "name" attribute) then that value will be
+            // returned. Otherwise a unique name will be constructed based on the frame
+            // parent hierarchy. The main (top-level) frame will always have an empty name
+            // value.
+            ///
+            /*--cef()--*/
+            virtual String^ GetName();
```

Comment:
I was thinking the same thing as well. Just made the first pass copy/paste. :smile: